apps/src/args.rs

```diff
@@ -298,6 +300,13 @@ impl Args for ClientArgs {
             None
         };
 
+        let source_port = args.get_str("--source-port");
+        let source_port = if !source_port.is_empty() {
+            source_port.parse::<u16>().unwrap()
+        } else {
+            0
+        };
```

Comment:
Since it has a default value, you don't need to call `is_empty()`.

```suggestion
        let source_port = source_port.parse::<u16>().unwrap();
```

Reply:
Good point, thanks